### PR TITLE
Console docs: edits to existing docs and stubbing out new docs for Is…

### PR DESCRIPTION
**Pull request by jenmalloy**
…sue 2408

### CODE REVIEW

File: documentation/modules/con-user-model.adoc [hunk 1]
```diff
@@ -1,11 +1,12 @@
 // Module included in the following assemblies:
 //
 // assembly-tenant-guide.adoc
+// master.adoc
 
 [id='con-user-model-{context}']
 = User model
 
-You can create users to access messaging clients and the {ProductName} Console through user definition files, which specify the authentication type and authorization policies.
+You can create users to access messaging clients through user definition files, which specify the authentication type and authorization policies.
```
Comment (k-wall) [inline documentation/modules/con-user-model.adoc]: Not relating to this change, but "create users to access messaging clients" is not clear.  Messaging clients (applications) connect using a messaging user,

Comment (jenmalloy) [reply documentation/modules/con-user-model.adoc]: @k-wall Thanks for pointing this out. Will fix this.

**Comment by jenmalloy** [comment]
@k-wall, @vbusch, Would you please take a look at the latest changes? Thanks!

File: documentation/modules/proc-create-address-space-console.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+// assembly-managing-address-spaces.adoc
+
+[id='proc-create-address-space-console-{context}']
+= Creating an address space using the {ProductName} Console
+
+You can create a new address space using the {ProductName} Console.
+
+.Procedure
+
+. Log in to the {ProductName} Console.
```
[the remark this answers is not in the document]
Comment (jenmalloy) [reply documentation/modules/proc-create-address-space-console.adoc]: @vbusch, Thanks. I was thinking about that after I used {ProductName}. I'll fix this

File: documentation/modules/proc-create-address-space-console.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+.Procedure
+
+. Log in to the {ProductName} Console. 
++
+For more information about how to access the {ProductName} Console, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#logging-into-console-messaging[Accessing the {ProductName} Console].
+
+. Click *Create*. The Create an instance wizard opens.
+
+. For the Component type, select *Address Space* and click *Next*.
```
[the remark this answers is not in the document]
Comment (jenmalloy) [reply documentation/modules/proc-create-address-space-console.adoc]: Fixed

File: documentation/modules/ref-example-roles.adoc [hunk 4]
```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// assembly-tenant-guide.adoc
+// using master.adoc
+
+[id='ref-example-roles-{context}']
+= {ProductName} resources and {ProductName} example roles
+
+The following table describes the {ProductName} resources.
```
Comment (k-wall) [inline documentation/modules/ref-example-roles.adoc]: I think we should split the table into two: resources that are of interest to the service administrator and those interest to the messaging tenant.

**Pull request merged**